### PR TITLE
[MINI-4458] bugfix: empty host locale value

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
Host locale is coming empty when the device language is Chinese. This PR aims to fix it with matching with spec. By default, if HostApp doesn't support any language, then it should return `en-US` to MiniApp.

## Links
- MINI-4458

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by Climbatize — APPROVED** [review]
LGTM

File: miniapp/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="miniapp.sdk.android.locale">ja-JP</string>
```
Comment (corycaywood) [inline miniapp/src/main/res/values-ja/strings.xml:2]: I think this should use `_` to match the other strings.

Comment (khairul-alam-licon) [reply miniapp/src/main/res/values-ja/strings.xml]: Thanks, I have updated it 🙇

**Review by munir-rakuten — APPROVED** [review]
LGTM 👍

**Pull request merged**